### ISSUE TITLE
Support v2 of cartodb's API

### ISSUE THREAD

**Issue by dimroc** [CartoDB/cartodb-rb-client#13]
All generated http requests seem to only hit v1 of the API, as evidenced in the vcr yml files.

**Comment by Ferdev** [comment]
Yeah, actually this is a known issue, but I didn't have time to fix it. The problem is that the ruby client uses both the SQL API (v2) and the REST API (v1) and at the very beginning of the project, I assumed that there'd be only one API.

I'd like to switch all the ruby client to the SQL API, but currently it's imposible since some API calls (like the create table one) can cause integrity problems, so that kind of calls must be performed using the REST API. We're working on fixing that issue in CartoDB, so I'll leave it as it is right now until we fix CartoDB and then, I'd switch the ruby client to use the SQL API completely and drop the REST API.

**Comment by tailic** [comment]
Ferdev, is this a Problem with CartoDB/SQL API or with this specific ruby client for cartodb?

**Comment by Ferdev** [comment]
Sorry tailic, I missed your message.

It's a problem with the ruby client, the SQL API is pretty fine :)
